_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1027,6 +1028,11 @@ locals {
   # TODO: Enter the default AWS region, the same as where the terraform state S3 bucket is currently provisioned.
   default_region = ""
 
+  # TODO: Fill these in after applying the account-baseline-root to the root account.
+  config_s3_bucket_name = ""
+  cloudtrail_s3_bucket_name = ""
+  cloudtrail_kms_key_arn = ""
```

Comment:
Couldn't you fill in the bucket names _now_?

Also, NIT: `fmt` the code :)

Reply:
Is this the recommended approach? AFAIK these are optional inputs to the account-baseline-root, which we don't guide them to set in this guide.